src/main/webapp/WEB-INF/tags/instructor/search/studentsSearchResults.tag

```diff
@@ -7,9 +7,12 @@
 <div class="panel panel-primary">
     <div class="panel-heading">
         <strong><jsp:doBody/></strong>
+        <div class='display-icon pull-right'>
```

Comment:
Sorry for these changes. Actually I was doing collapsing functionality for this before. After that realized that this was already corrected in other issue. And forgot to revert back these changes.